voxel51/users/api.py

```diff
@@ -293,6 +293,23 @@ def delete_analytic(self, analytic_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    def batch_get_analytic_details(self, analytic_ids):
+        '''Gets details about the analytics with the given IDs.
+
+        Args:
+            analytic_ids (list): the analytic IDs
+
+        Returns:
+            a dictionary mapping analytic IDs to dictionaries indicating
```

Comment:
These routes are a bit different than the other batch routes because they actually return data that the user wants. Would be nice to mention the name of the field where that info is stored here, and that the info is exactly what one would get from `get_analytic_details`

same for other methods and `api-js`

Reply:
Good point, how does https://github.com/voxel51/api-py/commit/3c7f3e7e7f868cfd97bda35fca50bb54f16447d4 look?